src/components/Home.tsx

```diff
@@ -68,7 +68,7 @@ export default function Home(): JSX.Element {
   useEffect(() => {
     let cancel = false;
     const populateUser = async () => {
-      const response = await getUser();
+      const response = await UserService.getUser();
```

Comment:
this is a terrible example since we should be using the user from the UserProvider here... but just to illustrate..
I didn't move the user service usage to UserProvider .. yet.. needs more work